bin/console

```diff
@@ -44,6 +44,12 @@ $application = new Application($kernel);
 $application->add(new \Console\Locale\ImportLocaleCommand());
 $application->add(new \Console\Thumbnails\GenerateThumbnailsCommand());
 if ($kernel->getContainer()->get('fork.settings') instanceof ModulesSettings) {
+    $application->add(
+        new \Console\Locale\CopyLocaleCommand(
```

Comment:
I'm not really sure why we're not using use statements here?

Reply:
Fixed